src/components/editions/headerImageCaption.tsx

```diff
@@ -79,14 +80,26 @@ const svgStyle = (iconColor?: string): SerializedStyles => css`
 `;
 
 interface Props {
-	caption: Option<string>;
+	caption: Option<DocumentFragment>;
 	credit: Option<string>;
 	styles?: SerializedStyles;
 	iconColor?: string;
 	iconBackgroundColor?: string;
 	isFullWidthImage: boolean;
 }
 
+const toReact = (node: Node, key: number): ReactNode => {
+	switch (node.nodeName) {
+		case 'EM':
+			return h('em', { key }, node.textContent);
```

Comment:
You're in a `tsx` file so you could do this if you prefer?

```tsx
<em key={key}>{node.textContent}</em>
```

Reply:
right you are, will switch that out